requirements.txt

```diff
@@ -10,6 +10,8 @@ djmail==0.11.0
 futures==3.0.3
 gunicorn==19.7.1
 lxml==4.1.0
+mock==2.0.0
+pbr==3.1.1
```

Comment:
What is this for?

Reply:
pbr is required by the mock package and the usage of the mock package is described in the comment above.